aries-site/src/data/structure.js

```diff
@@ -115,8 +118,28 @@ export const structure = [
     pages: [],
     pageLayout: 'plain',
   },
+  {
+    name: 'Design tokens',
```

Comment:
I think the space here is causing problems in the routing, need a modification to add in a `-`.

Resolves to `/design%20tokens` instead of `/design-tokens`

Reply:
Fixed.